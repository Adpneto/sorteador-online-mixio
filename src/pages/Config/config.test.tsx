import { render } from "@testing-library/react"
import { RecoilRoot } from "recoil"
import Configuration from "./config"

const mockNav = jest.fn()
jest.mock('react-router-dom', () => {
    return {
        useNavigate: () => mockNav
    }
})

describe('A pagina de configuração', () => {
    test('Deve ser renderizada corretamente', () => {
        const { container } = render(
            <RecoilRoot>
                <Configuration/>
            </RecoilRoot>
        )

        expect(container).toMatchSnapshot()
    })
})