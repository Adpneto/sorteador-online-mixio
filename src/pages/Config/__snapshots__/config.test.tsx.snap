// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A pagina de configuração Deve ser renderizada corretamente 1`] = `
<div>
  <div
    class="w-full h-screen flex flex-col items-center"
  >
    <div
      class="flex items-center h-[800px]"
    >
      <img
        alt="Logotipo da Mixio"
        class="w-40"
        src="logo.png"
      />
      <h1
        class="font-Shrikhand lg:text-[6rem] text-[3rem] text-[#261E40]"
      >
        Mixio
      </h1>
    </div>
    <div
      class="h-full w-full bg-[#261E40] rounded-t-xl flex flex-col items-center p-4"
    >
      <h2
        class="text-white font-bold text-[2rem] lg:text-[3rem] mt-6"
      >
        Vamos começar
      </h2>
      <h3
        class="text-white font-semibold text-sm lg:text-lg"
      >
        Criei uma lista com nome dos participante e depois sorteie!
      </h3>
      <form
        class=""
      >
        <div
          class="flex justify-center mt-8"
        >
          <input
            class="bg-[#B7BAC6] text-white w-full h-12 lg:w-96 lg:h-14 p-2 text-center rounded-l-xl font-bold text-base lg:text-xl"
            placeholder="Insira os nomes dos participantes"
            type="text"
            value=""
          />
          <button
            class="bg-[#6F74FF] w-40 h-12 lg:h-14 rounded-r-xl font-bold text-base lg:text-lg text-white"
            disabled=""
          >
            Adicionar
          </button>
        </div>
        
      </form>
      <ul
        class="m-5 flex flex-wrap"
      />
      <div>
        <button
          class="bg-[#6F74FF] px-4 py-4 rounded-lg text-white font-bold"
          disabled=""
        >
          Iniciar brincadeira
        </button>
      </div>
    </div>
  </div>
</div>
`;
